WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -404,6 +406,40 @@ public void run() {
         ActivityId.trackLastActivity(ActivityId.POST_EDITOR);
     }
 
+    // this method aims at recovering the current state of media items if they're inconsistent within the PostModel.
+    private void resetUploadingMediaToFailedIfNotInProgressOrQueued(Context context, PostModel post) {
+        boolean useAztec = AppPrefs.isAztecEditorEnabled();
+        if (useAztec && context != null && post != null) {
```

Comment:
I think we can get rid of some `if` nesting by doing a `return` if `useAztec` is false or there aren't any pending/in-progress uploads.